python/lsst/daf/butler/registries/sqlRegistry.py

```diff
@@ -267,6 +267,14 @@ def _makeDatasetRefFromRow(self, row, datasetType=None, dataId=None):
         return DatasetRef(datasetType=datasetType, dataId=dataId, id=row["dataset_id"], run=run,
                           hash=datasetRefHash, components=components)
 
+    def getAllCollections(self):
+        # Docstring inherited from Registry.getAllCollections
+        datasetCollectionTable = self._schema.tables["DatasetCollection"]
+        result = self._connection.execute(select([datasetCollectionTable.c.collection]).distinct()).fetchall()
```

Comment:
I started to dislike that single-letter `c` attribute, but it's OK to keep it for consistency with the rest of the code in this file.